lib/opam_build.ml

```diff
@@ -21,6 +21,11 @@ let opam_install ~variant ~upgrade_opam ~pin ~with_tests ~pkg =
         opam remove -y %s && opam %s%s %s
         res=$?
         test "$res" = 0 && exit 0
+        if test "$res" = 60 && diff -q /usr/bin/opam /usr/bin/opam-2.0; then
```

Comment:
```suggestion
        if test "$res" != 0 && diff -q /usr/bin/opam /usr/bin/opam-2.0; then
```
?? Can this also be hit with depext-related errors too?

Reply:
Testing seems to indicates that `opam-depext` also returns exit code 60 properly: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/597b4467d6799043fc07b4eafee2d7a038f95821/variant/compilers,4.04,mirage-conduit.3.0.0

Reply:
That wasn't quite what I meant - is solver timeout the only issue which can occur? I thought there were depext-related problems?

Reply:
ah, you're correct. Though, I would say I'd prefer to take one problem at a time. The depext issue is more difficult to detect and isolate.